actix-verify/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "actix"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# actix-multipart = "0.4"
```

Comment:
Please remove all the comments (in all the files) that don't help reader to understand the code.